CHANGELOG.md

```diff
@@ -89,6 +89,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - **CUMULUS-1690**
   - Removed Gulp/Browserify and their dependencies.
 
+### Fixed
+
+- **CUMULUS-1670**
+  - Fixed bug preventing update of providers
+
```

Comment:
This is in here twice now.  https://github.com/nasa/cumulus-dashboard/pull/656/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR77-R79